app/javascript/components/supplemental_application/SupplementalApplicationContainer.js

```diff
@@ -19,8 +19,19 @@ import { withContext } from './context'
 import StatusModalWrapper from '~/components/organisms/StatusModalWrapper'
 import validate from '~/utils/form/validations'
 
-const StatusUpdateSection = withContext(({ store }) => {
+const StatusUpdateSection = withContext(({ store, form, checkForValidationErrors }) => {
```

Comment:
Since these are being passed in here could we not just pass a prop of `isFormInvalid` or `isFormValid` with this check being run from the container component instead of calling it within these child components?